packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -205,6 +205,24 @@ class WebViewController {
     return _channel.invokeMethod('loadUrl', url);
   }
 
+  Future<bool> get canGoBack async {
```

Comment:
I wouldn't make `canGoBack` and `canGoForward` getters, as I feel like it hints that this is some cheap "give me back some value your maintaining", while this is actually making a method channel call.
I think a non getter method is a better fit here.